src/xcode/ENA/ENA/Source/Client/Client.swift

```diff
@@ -102,8 +112,23 @@ protocol Client {
 		day: String,
 		completion: @escaping HourCompletionHandler
 	)
+
+	// MARK: Getting the Configuration
+
+	typealias ExposureConfigurationCompletionHandler = (ENExposureConfiguration?) -> Void
+
+	/// Gets the remove exposure configuration. See `ENExposureConfiguration` for more details
+	/// Parameters:
+	/// - completion: Will be called with the remove configuration or an error if something went wrong. The completion handler will always be called on the main thread.
```

Comment:
♥️

Reply:
Thanks! Need to handover this ❤️ to the creator of this comment. (I just copied)